arch/riscv/Kconfig.isa

```diff
@@ -91,3 +93,55 @@ config RISCV_ISA_EXT_C
 	  RISC-V standard compressed instruction set extension, named "C",
 	  which reduces static and dynamic code size by adding short 16-bit
 	  instruction encodings for common operations.
+
+config RISCV_ISA_EXT_ZICSR
+	bool
+	help
+	  (Zicsr) - Standard Extension for Control and Status Register (CSR) Instructions
+
+	  The "Zicsr" extension introduces support for the full set of CSR
+	  instructions that operate on CSRs registers.
+
+config RISCV_ISA_EXT_ZIFENCEI
+	bool
+	help
+	  (Zifencei) - Standard Extension for Instruction-Fetch Fence
+
+	  The "Zifencei" extension includes the FENCE.I instruction that
+	  provides explicit synchronization between writes to instruction
+	  memory and instruction fetches on the same hart.
+
+config RISCV_ISA_BITMANIP_EXT_ZBA
```

Comment:
any reason this Kconfig symbol is prefixed `BITMANIP_` unlike other extensions that start with "Z" (e.g. `RISCV_ISA_EXT_ZICSR` and `RISCV_ISA_EXT_ZIFENCEI`)?

should it be just called `RISCV_ISA_EXT_ZBA` to be consistent with others? (i.e. we do not have `RISCV_ISA_EXT_CSR_ZICSR`)

Reply:
Because BitManip is indeed a different spec (these extensions are not even reported in the ISA manual). I don't have a strong opinion on this TBH, so I can definitely change those to the usual form.

Reply:
> Because BitManip is indeed a different spec (these extensions are not even reported in the ISA manual)

BitManip is actually in the current ratified ISA manual -- see p. 133 in the "Volume I: RISC-V Unprivileged ISA V20191213":

> Chapter 17. “B” Standard Extension for Bit Manipulation, Version 0.0
> This chapter is a placeholder for a future standard extension to provide bit manipulation instructions, including instructions to insert, extract, and test bit fields, and for rotations, funnel shifts,
and bit and byte permutations

It is just that, at the time of its release, the "B" extension was still WIP.

Reply:
done